src/cli/doctor.rs

```diff
@@ -24,6 +25,10 @@ impl Command for Doctor {
             }
         }
 
+        if let Some(latest) = cli::version::check_for_new_version() {
+            warn!("rtx version {} available", latest)
```

Comment:
this is a good addition, but let's change the messaging a bit
```suggestion
            warn!("new rtx version {} available, currently on {}", latest, env!("CARGO_PKG_VERSION"))
```